### ISSUE TITLE
Please check the number of Wasp Nests overmap extras being spawn

### ISSUE THREAD

**Issue by danielkwinsor** [CleverRaven/Cataclysm-DDA#52773]
### Describe the bug

I may be wrong, but the feeling while exploring the overmap is that there are many more wasp nasts extras than in previous versions.  It seems that Wasp Radio Towers are new, or at least they weren't seen by me.  A hypothesis is that while adding the Wasp Radio Towers someone bumped up the wasp spawn rate, or otherwise one affected the other.

### Steps To Reproduce

Start 0.F2 
Explore entire overmap.
Count Wasp Nests
Load 0.D game and repeat and compare
Alternatively, count number of search results in overmap, comparing against other map extras.

### Expected behavior

Fewer wasp nests.  I do not have empirical data, so if the current value is the expected value, that's ok.

### Versions and configuration

0.F2

**Comment by danielkwinsor** [comment]
And I forgot to say, if Wasp Radio Towers are new and the rate of Wasp Nests has not changed, ought the Wasp Nest rate to go down to account for the new wasps?

**Comment by Venera3** [comment]
Between 0.D and 0.F a second kind of wasp nest was added (the JSON-driven one for fields and such), and I did a bit of tweaking when adding the towers that sllightly overshot. They will be adjusted when I find the time or successfully con somebody into doing it for me.

**Comment by danielkwinsor** [comment]
Thanks for confirmation!

**Comment by JonathanLochridge** [comment]
So, did anyone actually get around to changing this? If not I wouldn't mind trying to fix it? Although, I am not sure how much it is supposed to be scaled back by?

**Comment by Night-Pryanik** [comment]
> If not I wouldn't mind trying to fix it?

Not at all, you're welcome to try.

**Comment by Venera3** [comment]
> Although, I am not sure how much it is supposed to be scaled back by?

I'd start by removing the map extra from the forest list - being surprised by them is a bit of a low punch, and they aren't nearly loud enough to warn you (IRL you can pretty often hear a hornet's nest in a quiet forest, and they aren't man-sized...). After that you could probably reduce the wasp house's spawn rate so that you find 1-2 in a medium-sized city. The field nests are pretty obvious, so you can usually work around them, I don't think they're much of a problem.

Balancing map extra spawn rates s is pretty annoying, either involving mathing it out and hoping or walking around with Debug Clairvoyance.
If you need any help feel free to @ me either here or on the devcord - you are kind of cleaning up after my stuff.

**Comment by JonathanLochridge** [comment]
Hmm, couldn't increasing their noise level also be an option? If that is an issue? Or reducing their occurrence in forests? As to me it doesn't seem like completely removing them from forests would be the best course. But I could try it. 

I don't mind working on it. I had been looking into getting to an easy map-related issue. I have some less easy ideas but I kind of want to ease into them.
Also, Debug clairvoyance? Is that using reveal map? Or something else?

**Comment by Venera3** [comment]
Their buzzing is handled by the footstep code, which has no way to freely define loudness and doesn't trigger on every move. The other option would be to move it over to the `PARROT` system, but I originally found that a bit too clunky (but I also never got around to looking into the footstep problem properly, so it's certainly an option).

Debug clairvoyance is a debug trait to grant you vision on everything in the reality bubble - bind a key to Debug Menu and go Debug Menu -> Player -> Mutate and search the list for it to gain it.

**Comment by JonathanLochridge** [comment]
Is the number of dermatics about right? Or should we also decrease them? Or maybe we could increase them to make them more common in comparison to wasps if people like them better?

**Comment by Venera3** [comment]
The lone spawns are probably not the problem (and I have some plans for them this cycle - a round of balancing is due on the girls anyway). 
The map extras are placed according to their frequency in `json/regional_map_settings.json` - each map extra list (`forest`) has a number of entries with their own weights and a general `freq` ( meaning in the forest's case 1-in-20 forests will spawn a map extra, chosen by a weighted roll from the below extras). Same for the wasp house in the list for city buildings, those two would be the numbers I'd start fiddling with.

Scattering some dermatiks around the forest works, yeah. They're pretty nonthreatening but kind of freak everybody out, so they're fun.

**Comment by JonathanLochridge** [comment]
huh, I don't know how I missed this. I figured out most of the content of the last post like 2ish hours ago. 
Although, the odds you quote seem off? Unless it's 1-in-20 forest tiles. Then in that case most of the larger forests I looked at were like 100-200 tiles? So 5-10 specials? Of which wasps shouldn't be that large a portion.
But since you like the idea I might increase the number of dermatiks in forests in particular up to the current value for normal wasps? And see if that makes the numbers line up with current experimental. 
I still need baseline numbers for how many someone can find in about 10-30 minutes. I have the changed values. Although, I can do that myself if I get enough other tasks done. But, I am about done doing testing for today. I brought up a few smaller issues in the draft PR if you want to check it out.
